source/packages/services/assetlibrary-export/src/etl/etl.service.ts

```diff
@@ -20,34 +20,36 @@ import { ExtractService } from './extract.service';
 import { TransformService } from './transform.service';
 import { LoadService, Loaded } from './load.service';
 import { S3Utils } from '../utils/s3.util';
+import {LabelsService} from '../labels/labels.service';
 
 @injectable()
 export class ETLService {
     constructor(
         @inject(TYPES.ExtractService) private extractService: ExtractService,
         @inject(TYPES.TransformService) private transformService: TransformService,
         @inject(TYPES.LoadService) private loadService: LoadService,
+        @inject(TYPES.LabelsService) private labeslService: LabelsService,
         @inject('aws.s3.export.bucket') private exportBucket: string,
         @inject('aws.s3.export.prefix') private exportKeyPrefix: string,
         @inject(TYPES.S3Utils) private s3Utils: S3Utils
     ) {}
 
     public async processBatch(batchId: string): Promise<Loaded> {
+        logger.debug(`ETLService: processBatch in:`);
```

Comment:
Just out of curiosity, not suggesting a change: What's the reasoning behind dropping the batch info from the log line here?

Reply:
Great Question, on this particular line, batchId could be added back because it will be printed once.  However, I removed logs from some other places where there was tons of logs being generated, resulting in tons of cloudwatch IO. It also puts memory overhead on the lambdas as well. I removed some logs to optimize the processing of these batches. The batches could have 1000 records to process, hence the change to optimize memory utilization and IO.

Reply:
if it's useful in certain situations, you could always change the detailed logging to `silly`.

Reply:
The reason I asked on this line specifically is that it logs before `batch.range` gets expanded to `batch.items`, so should be less verbose than what currently goes into logs.